crates/python/src/lib.rs

```diff
@@ -36,6 +36,7 @@ create_init_submodule! {
     submodules: [
         "client": client::init_submodule,
         "compiler": compiler::init_submodule,
+        "grpc": grpc::init_submodule,
```

Comment:
I know you are just carrying forward with an existing module and just actually exporting it here, and I know the plan is to add a wrapped type later, but I don't love exporting raw bindings to the API here, and especially exposing them as a "GRPC" module to users.

Other modules get coupled to it because by it's very nature it contains models that requests across the stack need, but the individual models are typically scoped to a single kind of request, so why have them all co-located? It's also ambiguous when you need to pull it in, since our services use a mix of REST and GRPC.

To me, it feels like we're exposing unnecessary implementation details to users and making it more complex than needed to reason about our API and how to use it. `translation` is a clean concept within the scope of our product but `grpc` is ambiguous and doesn't help users discover useful things about our API. 

I could be convinced to re-export the needed GRPC model directly from the translation module, that way it is at least grouped within a concept, reducing the number of imports needed and making it easier to find the needed type with tooling.